app/models/survey.rb

```diff
@@ -14,6 +14,8 @@ class Survey < ActiveRecord::Base
   has_many :question_orders
   has_many :survey_responses
 
+  has_many :questions, through: :question_orders
```

Comment:
Why did we have `survey_id` on `question_orders` model instead of `questions`? Conceptually, it makes more sense to me for `questions` to have `survey_id` directly.

I'll leave this up to you to decide.